dll/cpl/mmsys/lang/pt-PT.rc

```diff
@@ -13,7 +21,7 @@ BEGIN
     LTEXT "Fraco", IDC_VOLUME_LOW, 62, 66, 30, 9
     LTEXT "Forte", IDC_VOLUME_HIGH, 195, 66, 30, 9
     AUTOCHECKBOX "&Sem som", IDC_MUTE_CHECKBOX, 50, 78, 140, 10
-    AUTOCHECKBOX "Colocar um &icone de volume na barra de tarefas", IDC_ICON_IN_TASKBAR, 50, 92, 190, 10
+    AUTOCHECKBOX "Colocar um í&cone de volume na barra de tarefas", IDC_ICON_IN_TASKBAR, 50, 92, 185, 10
     PUSHBUTTON "A&vançadas...", IDC_ADVANCED_BTN, 150, 105, 75, 15
```

Comment:
"A&vançadas..." collides with "&Volume dos altifalantes...". Please fix that since you touched the file anyway.